README.md

```diff
@@ -1 +1,79 @@
-# solution-pack-malware-response-using-siem-and-edr-solutions
+# Malware Response Using SIEM & EDR Solutions
+
+## Release Information
+- Solution Pack Version: 1.0.0
+- Minimum Compatible FortiSOAR™ Version : 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Overview
+### Introduction
+This article describes the Malware Response Using SIEM & EDR Solutions Pack. The solution pack demonstrates the scenarios and use cases around Malware reported by ElasticSearch or Splunk.
+
+Configure ingestion using ElasticSearch or Splunk connector. Ingestion process creates an alert of type 'Malware', and then manually triggers the response workflow.
+
+Refer to Simulation Scenarios - "Malware Response Using SIEM and EDR Solutions (ElasticSearch / Active Directory / VMware vSphere)" or "Malware Response Using SIEM and EDR Solutions (Splunk/QRadar/FortiEDR)" to experience the use case without ElasticSearch or Splunk connector configuration.
```

Comment:
what about Qradar config?

Reply:
Alert coming from either ElasticSearch or Splunk.
Here Qradar is not used for ingestion it is used for threat hunting